test/DynamoCoreUITests/VisualizationManagerUITests.cs

```diff
@@ -122,7 +122,7 @@ public void VisualizationInSyncWithPreview()
             //now flip off the preview on one of the points
```

Comment:
`IsVisible` cannot be set directly now, so go through backdoor API `ModelBase.UpdateValue` method.